xlgui/widgets/queue.py

```diff
@@ -50,10 +50,10 @@ def _saveas_playlist_cb(widget, name, page, context):
             exaile.playlists.save_playlist(pl)
             page.container.create_tab_from_playlist(pl)
     
-    items.append(smi('saveas', ['clear'], None, 'gtk-save-as',
+    items.append(smi('saveas', ['clear'], _("_Save as Playlist"), 'document-save-as',
         _saveas_playlist_cb))
     items.append(sep('tab-close-sep', ['saveas']))
-    items.append(smi('tab-close', ['tab-close-sep'], None, 'gtk-close',
+    items.append(smi('tab-close', ['tab-close-sep'], _("C_lose Tab"), 'window-close',
```

Comment:
See note above:
I've changed it the other way: `_Close Tab` → `Close _Tab` since I need `_C` for "Clear"